packages/mdl-theme/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "mdl-theme",
+  "version": "1.0.0",
+  "description": "The Material Design Lite theme component",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "mdl-typography": "^1.0.0"
```

Comment:
I think you can remove this from dev deps since it's used in the demo, which isn't included in this package.